scss/atoms/typography/_functions.scss

```diff
@@ -11,6 +11,6 @@
 * 1. font-size: em(14px);
 * 2. font-size: em(30px/14px);
 */
-@function em($value, $context: map-get($typography-font-default, font-size)) {
-	@return ($value / $context * 1em);
+@function rem($value, $context: map-get($typography-font-default, font-size)) {
+	@return ($value / $context * 1rem);
```

Comment:
We should reconsider this approach of deciding in pixels and then converting to ems/rems. It should be the other way round, deciding a scale then converting it to pixels(only if necessary). I'll bring it up in next meeting. We can keep the PR pending until then.